boat_visualizer/lib/boat_visualizer_web/live/map_live.ex

```diff
@@ -14,29 +16,122 @@ defmodule BoatVisualizerWeb.MapLive do
     Animation.set_map_view(map_center)
     Animation.set_marker_coordinates(initial_coordinates)
 
-    {:ok,
-     socket
-     |> assign(:current_coordinates, initial_coordinates)
-     |> assign(:coordinates, coordinates)
-     |> assign(:ranged_coordinates, coordinates)
-     |> assign(:map_center, map_center)
-     |> assign(:min_position, 0)
-     |> assign(:max_position, Enum.count(coordinates) - 1)
-     |> assign(:current_position, 0)
-     |> assign(:current_min_position, 0)
-     |> assign(:current_max_position, Enum.count(coordinates) - 1)
-     |> assign(:show_track, true)}
+    {now, us} = NaiveDateTime.utc_now() |> NaiveDateTime.to_gregorian_seconds()
+    now = now + us / 1_000_000
+
+    min_lat = 42.1666
+    max_lat = 42.4093
+    min_lon = -71.0473
+    max_lon = -70.8557
```

Comment:
Is this something we could calculate dynamically? Perhaps the way `Animation.set_map_view/1` could be used as reference for that. Either way, let's have it as something to tackle in the future once we add the ability to support other csv besides the one we have hardcoded

Reply:
We have the base code for this already, whenever the map changes I send an event updating the bounding box. We'd just have to associate it with map load as well

Reply:
Should be an easy update, but we should probably leave this for a follow up PR